e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -243,6 +243,24 @@ describe('I18456 Built-in slash commands: common', () => {
                 });
         });
     });
+
+    it('MM-T2834 Slash command help stays visible for system slash command', () => {
+        // # Login as user 1 and visit default channel
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+
+        // # Type the rename slash command in textbox
+        cy.get('#post_textbox', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible').clear().type('/rename ');
+
+        // # Scan inside of suggestion list
+        cy.get('#suggestionList').should('exist').and('be.visible').within(() => {
```

Comment:
Minor: For positive assertion chains, `exist` is redundant and can be removed if `be.visible` is present. Also prefer `be.visible` over `exist` on the other verifications.
```suggestion
        cy.get('#suggestionList').should('be.visible').within(() => {
```

Reply:
Hi,

I believe both should be present as `exist` determines if it exists in the dom and `visible` checks if it visible to user and not scrolled hidden. Hence i have written both the assertions.

C.C @saturninoabril

Reply:
Hi @M-ZubairAhmed , I think you can infer implicit existence on visibility (positive assertion), i.e. an element cannot be visible if it doesn't exist. Existence check is required on negative assertions though. It's not a big deal, I'll go ahead and approve the PR. Thanks for the contribution!!